lib/shoryuken/manager.rb

```diff
@@ -12,7 +12,17 @@ class Manager
 
     def initialize(condvar)
       @count = Shoryuken.options[:concurrency] || 25
-      raise(ArgumentError, "Concurrency value #{@count} is invalid, it needs to be a positive number") unless @count > 0
+      @fetcher_pause_interval = Shoryuken.options[:fetcher_pause_interval] || 1
```

Comment:
we are using two different symbols: 
`:dispatch_interval` and `:fetcher_pause_interval`